examples/mongodb/README.md

```diff
@@ -0,0 +1,88 @@
+# Overview
+
+OpenTelemetry Mongodb Instrumentation allows the user to automatically collect trace data and export them to the backend of choice (we can use Zipkin or Jaeger for this example), to give observability to distributed systems.
+
+This is a modification of the HTTP example that executes multiple parallel requests that interact with a Mongodb server backend using the `mongo` npm module. The example displays traces using multiple connection methods.
+- Create Collection Query
+- Insert Document Query
+- Fetch All Documents Query
+
+### Http Plugin Options
```

Comment:
This should be inside the plugin readme ([there](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/master/plugins/node/opentelemetry-plugin-mongodb/README.md)), not the example. Also the option of the plugin only contains `enhancedDatabaseReporting` with your PR, why did you add the others ?

Reply:
Moving this. 
I thought it would be good to have all available options documented in one place, and because the plugin takes all of these as input.

Reply:
@vmarchaud should i only add `enhancedDatabaseReporting` ?

Reply:
@romil-punetha yeah, we generally document only the custom options.

Reply:
Ok. Editing that.